--- a/system/etc/init/hw/init.rc
+++ b/system/etc/init/hw/init.rc
@@ -13,6 +13,7 @@
     setprop sys.usb.configfs 0

 on init
+    start selinux-permissive
     export ANDROID_ROOT /system
     export ANDROID_DATA /data
     export EXTERNAL_STORAGE /sdcard
@@ -231,3 +231,10 @@
     symlink /config/usb_gadget/g1/functions/ffs.fastboot /config/usb_gadget/g1/configs/b.1/f1
     write /config/usb_gadget/g1/UDC ${sys.usb.controller}
     setprop sys.usb.state ${sys.usb.config}
+
+service selinux-permissive /system/bin/selinux-permissive
+    disabled
+    oneshot
+    user root
+    group root
+    seclabel u:r:recovery:s0
